index.html

```diff
@@ -138,8 +147,23 @@
 
         function onModelChange() {
             var extension = options.folder.indexOf("Binary") !== -1 ? ".glb" : ".gltf";
-            load("models/2.0/" + options.model + "/" + options.folder + "/", options.model + extension);
+
+            // maps to attribution for image files here per cc license guidelines
+            // first element must match background name in gui menu
+            var imageSources = {
+                "country": "",
+                "wobblyBridge": "https://hdrihaven.com/bundle.php?b=free_bundle"
+            };
+
+            if (imageSources[options.background]) {
+                options.attribution = imageSources[options.background];
+            }
+
+            load("models/2.0/" + options.model + "/" + options.folder + "/",
+                  options.model + extension,
+                  "images/" + options.background + ".babylon.hdr"); 
```

Comment:
maybe create a `var hdrExtension = ".babylon.hdr"` and then use that here, just like `extension` is being used right above?